consensus/bor/heimdallapp/checkpoint.go

```diff
@@ -0,0 +1,50 @@
+package heimdallapp
+
+import (
+	"context"
+	"encoding/json"
```

Comment:
Could you use go-iterator instead?

Reply:
Okay

Reply:
and in other places is the same

Reply:
Are you talking about this one ? https://pkg.go.dev/google.golang.org/api/iterator

Reply:
I think you were talking about this, My bad
https://pkg.go.dev/github.com/json-iterator/go

Reply:
that one